lib/AArch64Base.ml

```diff
@@ -607,8 +607,7 @@ let tr_simd_variant = function
   | VSIMD8 -> MachSize.Byte
   | VSIMD16 -> MachSize.Short
   | VSIMD32 -> MachSize.Word
-  | VSIMD64 -> MachSize.Quad
-  | VSIMD128 -> MachSize.S128
+  | VSIMD64 | VSIMD128 -> MachSize.Quad
```

Comment:
This change looks counter-intuitive, if appropriate, some comment may prevent reverting the change.

Reply:
Thanks, I have reverted this so that it now returns `MachSize.S128` for `VSIMD128`. I believe that this change was initially made to indicate that a 128-bit memory access is made up of 2x64-bit accesses, but it's now clear that `MachSize.Quad` was the wrong value to return given our implementation (we use `tr_simd_variant` to work out the access size for many of the scalar SIMD instructions).